src/main/webapp/projects.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
```

Comment:
Indentation: should this be two spaces to the right?

Reply:
Yup! Good catch. Fixing.